REQUIRE

```diff
@@ -2,4 +2,3 @@ julia 0.3
 Mustache
```

Comment:
Had dinner with some continuum people last night so was reminded of this and thought I'd see what was new